packages/components/src/components/sequence/ReturnAction.vue

```diff
@@ -163,6 +163,11 @@ export default {
       type: Array,
       required: true,
     },
+    appMap: {
+      type: Object,
+      required: true,
+      readonly: true,
+    },
```

Comment:
Same comment here regarding too much data passed to this component. A return value string would be ideal.

Reply:
updated.